src/components/about/aboutCard.styles.jsx

```diff
@@ -0,0 +1,24 @@
+import tw, { css } from 'twin.macro'
+import styled from 'styled-components'
+
+export const Container = styled.div(({ bgColor }) => [
+  tw`rounded-[4.4rem] grid place-items-center py-16 px-10 brk-6:w-[90%]`,
+  css`
+    background-color: ${bgColor};
+  `
+])
+
+export const Section = styled.div`
+  ${tw`text-white-1 flex items-center justify-center flex-col`}
+`
+export const Title = styled.h1(({ isSuperTitle }) => [
+  isSuperTitle
+    ? tw`brk-8:text-[96px] brk-7:text-[96px] text-[60px] mt-20`
```

Comment:
make the title font smaller for screens smaller than 538px. 
![image](https://github.com/dscnitrourkela/project-vanilla/assets/68679980/5516ebb8-8eb6-41f6-ac74-e9690ac834df)